CMakeLists.txt

```diff
@@ -1134,7 +1134,9 @@ option(ZLIB_ENABLE_TESTS "Build test binaries" ON)
 if(ZLIB_ENABLE_TESTS)
     enable_testing()
 
-    add_subdirectory(test)
+    if("${CMAKE_VERSION}" VERSION_GREATER_EQUAL "3.12")
```

Comment:
No need for double quotes in either side of the equation. At the top of CMakeLists.txt we use something like:
```
if(CMAKE_VERSION VERSION_LESS 3.12)
```

Reply:
Changed.